packages/mini-browser/src/browser/mini-browser-open-handler.ts

```diff
@@ -243,8 +243,8 @@ export class MiniBrowserOpenHandler extends NavigatableWidgetOpenHandler<MiniBro
         return uri;
     }
 
-    protected async openUrl(): Promise<void> {
-        const url = await this.quickInputService.open({
+    protected async openUrl(arg: string | undefined): Promise<void> {
```

Comment:
`arg?: string` should be optional, for backward compatibility

Reply:
fixed, thank you!